exporter/datadogexporter/internal/translator/metrics_translator_test.go

```diff
@@ -106,12 +108,24 @@ func (t testProvider) Hostname(context.Context) (string, error) {
 	return string(t), nil
 }
 
-func newTranslator(t *testing.T, logger *zap.Logger) *Translator {
-	tr, err := New(
-		logger,
+func newTranslator(t *testing.T, logger *zap.Logger, resourceAttributesAsTags, instrumentationLibraryMetadataAsTags bool) *Translator {
```

Comment:
nit: maybe adding a `options ...Options` argument list instead is better to avoid having to change the function signature for every new thing that we want to test